frontend_tests/node_tests/templates.js

```diff
@@ -2,9 +2,12 @@
 
 const {strict: assert} = require("assert");
 
-const {zrequire} = require("../zjsunit/namespace");
+const {JSDOM} = require("jsdom");
+
+const {set_global, zrequire} = require("../zjsunit/namespace");
 const {run_test} = require("../zjsunit/test");
 
+set_global("DOMParser", new JSDOM().window.DOMParser);
```

Comment:
Why do we need JSDOM for this?  I would have expected it to be unnecessary since we're just transmuting/comparing strings.

Reply:
Because the `rendered_markdown` Handlebars helper uses `clean_user_content_links` which uses `DOMParser`. This logic should perhaps be moved out of Handlebars land though.

Reply:
Is this something that is out of the scope of this PR?  And if so, should we raise an issue for follow-up?

Reply:
Yes.